site/zenodo_rdm/github/release.py

```diff
@@ -79,4 +79,24 @@ def metadata(self):
             if citation_metadata:
                 output.update(citation_metadata)
 
+        if not output.get("creators"):
```

Comment:
This is duplicated from `invenio-rdm-records` but the way we structured the code forces this. We'd need a rework on how the metadata is computed, maybe something for later.